extensions/scheduler/runtime/src/main/java/io/quarkus/scheduler/Scheduler.java

```diff
@@ -13,11 +13,21 @@
      */
     void pause();
 
+    /**
+     * Pause a specific trigger.
+     */
```

Comment:
I would add a link to `@Scheduled.identity()` so that it's clear what the identity means here..

Reply:
added the link